.pre-commit-config.yaml

```diff
@@ -2,7 +2,6 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.4.0
     hooks:
-      - id: trailing-whitespace
```

Comment:
This did cover files other than rust files, so rustfmt isn't a perfect replacement.

But I agree the case is a tough one; not sure there's a good solution. Let's see if things creep in...

Reply:
I am pretty sure `prettier` gets rid of trailing whitespace in most other file formats.

Reply:
Can we exclude only Rust files with `exclude`?
https://pre-commit.com/#adding-pre-commit-plugins-to-your-project

Reply:
Good idea @eitsupi ; done!